one_fm/templates/pages/applicant-docs.html

```diff
@@ -4,101 +4,236 @@
+                                    <br>
+                                </div>
                             </div>
-                            <p>Extracting Text from your civil id, We'll Redirect You as soon as it is done</p>
-                        </div>
-                        <div class="file-upload">
-                            <button class="file-upload-btn" type="button"
-                                onclick="$('.file-upload-input').trigger( 'click' )">Add Image</button>
-                            <div class="image-upload-wrap">
-                                <input class="file-upload-input" type='file' onchange="readURL(this);"
-                                    accept="image/*" />
-                                <!-- <div class="drag-text">
-                                    <h3>Drag and drop your Civil ID</h3>
-                                </div> -->
+                            
+
+                            
+                            
+                            <!--
+
+                            <div class="row">
+                                    <div class="form-group col-md-6">
+                                        <label  for="file">Upload Front Side of your passport:</label>
+                                        <input class="form-control input" type="file" id="front_passport" placeholder="Front Passport" onchange="front_passport_extract(this)" name="file"  accept="image/png, image/jpeg" >
+                                    <div class="custom-control custom-checkbox checkbox-xl">
+                                        <input type="checkbox" class="custom-control-input" id="noPassport" >
+                                        <label class="custom-control-label" for="noPassport">No Passport</label>
+                                    </div>
+                                    </div>
+                                    <div class="form-group col-md-6">
+                                        <label  for="file">Upload Back Side of your passport:</label>
+                                        <input class="form-control input" type="file" id="back_passport" placeholder="Back Passport" onchange="back_passport_extract(this)" name="file"  accept="image/png, image/jpeg" >
+                                        <br>
+                                    </div>
                             </div>
-                            <div class="file-upload-content">
-                                <img class="file-upload-image" src="#" alt="your image" />
-                                <div class="image-title-wrap">
-                                    <button type="button" onclick="removeUpload()" class="remove-image">Remove <span
-                                            class="image-title">Uploaded Image</span></button>
-                                </div>
+
+                            -->
```

Comment:
- Remove white space 
- Remove commented codes if we dont needed

Reply:
@pjamsheer , I have resolved this in the next story.